wagtail/admin/templates/wagtailadmin/shared/human_readable_date.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags i18n %}
+
+<button type="button" class="human-readable-date" data-tippy-content="{{ date|date:"DATETIME_FORMAT" }}">
```

Comment:
```suggestion
<button class="w-human-readable-date"  type="button" data-tippy-content="{{ date|date:"DATETIME_FORMAT" }}">
```

Adding W prefix and moving type attribute to after class for readability.